docs/api.yml

```diff
@@ -0,0 +1,59 @@
+openapi: "3.0.0"
+info:
+  title: Evidence Store API
+  version: 1.0.0
+paths:
+  /metadata:
+    post:
+      operationId: save-metadata
+      summary: Saves metadata to S3 bucket
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+      responses:
+        201:
+          description: Saved
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/metadata"
+              example:
+                {
+                  "documentId": "kjsak2",
+                  "firstName": "The name",
+                  "dob": "1999-01-01",
+                }
+  /{documentId}:
+    get:
+      operationId: get-metadata
+      summary: Gets a document from S3 using the document id
+      parameters:
+        - $ref: "#/components/parameters/documentId"
+      responses:
+        200:
+          description: OK
```

Comment:
what do people generally put for swagger descriptions? (i have no idea). In the save metadata one you have "Saved", OK feels a little less descriptive

Reply:
It always seems like a bit of an awkward field for responses, I usually just go with the HTTP status message that goes along with the status code unless there's something particularly interesting about the response; but I can't say I have particularly strong reasoning for doing so.

Reply:
So you'd go with "Created" for the 201?

Reply:
Yeah, 9 times out of 10.
I've seen people describe what is in the response as well though like "newly created metadata".